### ISSUE TITLE
How to get tags from user input?

### ISSUE THREAD

**Issue by marttt** [anoved/Pinboard-Recipe#2]
Hi! I'm interested in only fetching articles that contain a particular tag. It's easy to add the `tags` parameter into the code, but how could one pass this argument from the command line?

The eventual command might then be something like `ebook-convert Pinboard.recipe "tags-from-user-input" --title=Title`.

I know it should be something fairly simple with `str(sys.argv[1]`, but considering the recipes are further handled by ebook-convert, I'm confused and, being a non-programmer, haven't got it to work thus far.

Thanks for any help/workaround,
and for providing this recipe in the first place!

**Comment by anoved** [comment]
Good question. To be honest, I only ever use these recipes through the Calibre GUI, so I'm not sure whether `ebook-convert` can pass command line arguments to the recipe script.

If you only use a few tags, one workaround might just be to make a few copies of the recipe, each hard-coded to retrieve a different tag.

**Comment by marttt** [comment]
Yes, that's exactly what I did for now, but looks like I need more
flexibility, considering I also use Pinboard to collect notes/links for my
running projects.
I gues I'll ask the people in mobileread.com forums for some advice. Thanks
very much for your reply!

2013/12/30 Jim notifications@github.com

**Comment by anoved** [comment]
No prob. If you find a solution, please do come back and share it!

**Comment by marttt** [comment]
Sure, no problem!

2013/12/30 Jim notifications@github.com
